nodejs/week1/warmup.js

```diff
@@ -0,0 +1,25 @@
+class Circle {
+   constructor(radius) {
+      this.radius = radius;
+   }
+   getDiameter() {
+      return this.radius * 2;
+   }
+   getCircumference() {
+      return 2 * Math.PI * this.radius;
+   }
+   getArea() {
+      return Math.PI * Math.pow(this.radius, 2);
+   }
+}
+
+const circle = new Circle(10);
+const diameter = circle.getDiameter();
+console.log(diameter);
+
+const ference = circle.getCircumference();
```

Comment:
Not a mistake, but naming it `ference` is a bit odd :) Not sure if it's a real word